test/unit/constants.dart

```diff
@@ -152,7 +152,7 @@ final List<String> commonSkippedTests = []
   ..addAll(skippedRelativeJsonPointerFormatTest)
   ..addAll(skppedIdnHostnameFormatTests);
 
-final List<String> draft9SkippedTestFiles = [
```

Comment:
Thanks for cleaning this up! I noted it, gritted my teeth, and just moved on.

Reply:
:scream: Thanks!